conjur/errors.py

```diff
@@ -50,3 +50,6 @@ class CertificateHostnameMismatchException(Exception):
     def __init__(self):
         self.message = MISMATCH_HOSTNAME_MESSAGE
         super().__init__(self.message)
+
+class ConfigurationMissingException(Exception):
+    """ Exception for when configuration is missing """
```

Comment:
Should there be __init__ function inside the class?

Reply:
added.